### PR TITLE
Create BPFDoor_abnormal_process_id_or_lock_file_accessed.yml

### PR DESCRIPTION
**Pull request by Zandmann**
detection for BPFDoor IoC files run from temporary file storage

### CODE REVIEW

**Review by Zandmann — COMMENTED** [review]
file name changed
MITRE edited

**Comment by frack113** [comment]
Hi, rule yml filename is lowercase

**Review by Zandmann — COMMENTED** [review]
Hi, changed to lowercase

File: rules/linux/auditd/lnx_auditd_bpfdoor_file_accessed.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            - /var/run/haldrund.pid
+            - /var/run/xinetd.lock
+            - /var/run/kdevrund.pid
+    condition: selection
+tags:
+   - attack.execution
+   - attack.t1106
+   - attack.t1059
+falsepositives:
+   - Less Likely
```
Comment (nasbench) [inline rules/linux/auditd/lnx_auditd_bpfdoor_file_accessed.yml:26]: Can you please change this to `Unlikely` to be conform with other rules :D 
Thanks

**Review by Zandmann — COMMENTED** [review]
changed to 'Unlikely' ;)

**Pull request merged**